_layouts/bib.html

```diff
@@ -29,76 +29,65 @@
           <div class="author">
           {% assign author_array_size = entry.author_array | size %}
 
+          {% assign author_array_limit = author_array_size %}
           {%- if site.max_author_limit and author_array_size > site.max_author_limit %}
             {% assign author_array_limit = site.max_author_limit %}
-          {% else %}
-            {% assign author_array_limit = author_array_size %}
           {% endif %}
 
           {%- for author in entry.author_array limit: author_array_limit -%}
             {%- assign author_is_self = false -%}
-            {%- if author.last == site.scholar.last_name %}
+            {%- assign author_last_name = author.last | remove: "*" -%}
```

Comment:
I think other symbols such as `^` should also be removed.

Reply:
Thank you for the reminder. I took the ^ symbol as a math sign unconsciously. 
I have referred to the guideline by [PLOS ONE](https://journals.plos.org/plosone/s/submission-guidelines), in particular this [PDF document](https://journals.plos.org/plosone/s/file?id=3fac/PLOS%20Affiliations%20Formatting%20Guidelines.pdf).
The guideline suggested five common symbols for indicating special authorships.